woocommerce/woocommerce
# Core Profiler - Add obw/core-profiler to the extensions endpoint

### Changes proposed in this Pull Request:

Closes #38408 .

This PR adds `obw/core-profiler` entry to the free-extensions endpoint. In addition to the new `obw/core-profiler`, it also includes the following changes.

1. Added `name` field to all plugins.
2. Added an icon for `jetpack` and `woocommerce-services`

I'll work on WCCOM changes as well.

### How to test the changes in this Pull Request:


1. Make sure `woocommerce_show_marketplace_suggestions` is set to `no`
2. Change this [line](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/src/Admin/API/OnboardingFreeExtensions.php#L66) to return `true`
3. Using curl or Postman, access `/wp-json/wc-admin/onboarding/free-extensions`
4. The response should have `obw/core-profiler`
5. Install and activate one of the plugins from `obw/core-profiler`
6. Access the endpoint again
7. Confirm the plugin is still included in `obw/core-profiler`
